RFCs/0018-Skip-Default-Extensions.md

```diff
@@ -0,0 +1,42 @@
+# 0018 Skip Default Extensions
+
+## Summary
+This note outlines the proposed changes for adding option for skipping lookup and initialization of default extensions.
```

Comment:
By "extensions" you mean "adapters" in particular, right? Please use "adapters" so it is clear - extensions could mean loggers/datacollectors etc.

Reply:
Renamed to adapters